File: android/app/src/main/java/tech/logica10/soniclair/NotificationBroadcastReceiver.java
package tech.logica10.soniclair;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;

public class NotificationBroadcastReceiver extends BroadcastReceiver {

    public NotificationBroadcastReceiver() {

    }

    private static final String TAG = "NotificationBroadcastReceiver";

    @Override
    public void onReceive(Context context, Intent intent) {
        Globals.NotifyObservers(intent.getAction(),null);
    }
}

File: android/app/src/main/kotlin/tech/logica10/soniclair/NotificationBroadcastReceiver.kt
package tech.logica10.soniclair

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent
import android.util.Log

class NotificationBroadcastReceiver : BroadcastReceiver() {
    override fun onReceive(context: Context, intent: Intent) {
        Log.i("SonicLair", "Recevied intent with action " + intent.action)
        Globals.NotifyObservers(intent.action, null)
    }
}